app/main/helpers/email.py

```diff
@@ -49,5 +50,12 @@ def decode_email(token):
     ts = URLSafeTimedSerializer(main.config["SECRET_KEY"])
     decoded = ts.loads(token,
                        salt=main.config["RESET_PASSWORD_SALT"],
-                       max_age=86400)
+                       max_age=86400, return_timestamp=True)
     return decoded
+
+
+def token_created_before_password_last_changed(token_timestamp, user):
+
+        password_last_changed = datetime.strptime(
+            user['users']['passwordChangedAt'], '%a, %d %b %Y %H:%M:%S %Z')
```

Comment:
The Created At and Updated at fields in the API specify the following format: `DATE_FORMAT = "%Y-%m-%dT%H:%M:%S%Z"` - I think we should probably use the same format for all our datetime strings.